main.go

```diff
@@ -14,13 +17,40 @@ var (
 
 func main() {
 	app := cli.NewApp()
-	app.Name = "rancher-vxlan"
 	app.Version = VERSION
 	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "listen",
+			Value: ":8111",
+		},
 		cli.BoolFlag{
 			Name:   "debug, d",
 			EnvVar: "RANCHER_DEBUG",
 		},
+		cli.BoolFlag{
+			Name:   "disable-tx-offload",
```

Comment:
This is not needed. I prefer to get rid of this option.
But some network adapters can support VXLAN Overlay HW Offloading.
e.g. : https://software.intel.com/en-us/blogs/2015/01/29/optimizing-the-virtual-networks-with-vxlan-overlay-offloading

Created a new issue https://github.com/rancher/rancher/issues/9681 as an enhancement.